lib/matplotlib/ticker.py

```diff
@@ -1016,33 +1048,29 @@ def __call__(self, x, pos=None):
         else:
             base = '%s' % b
 
-        if coeff in self.sublabel:
-            if not is_decade and self.labelOnlyBase:
-                return ''
-            elif not is_decade:
-                return self._non_decade_format(sign_string, base, fx, usetex)
-            else:
-                if usetex:
-                    return (r'$%s%s^{%d}$') % (sign_string,
-                                               base,
-                                               nearest_long(fx))
-                else:
-                    return ('$%s$' % _mathdefault(
-                        '%s%s^{%d}' %
-                        (sign_string, base, nearest_long(fx))))
-        else:
+        if self.labelOnlyBase and not isDecade:
             return ''
+        if self._sublabels is not None and coeff not in self._sublabels:
+            return ''
+
+        if not is_decade:
```

Comment:
Yes, thanks for catching that.  It looks like every instance of `is_decade` as a scalar, as opposed to a function, should be replaced by `isDecade`.